taskvine/src/bindings/python3/ndcctools/taskvine/dask_executor.py

```diff
@@ -219,7 +235,11 @@ def _dask_execute(self, dsk, keys):
         indices = {k: inds for (k, inds) in find_dask_keys(keys)}
         keys_flatten = indices.keys()
 
-        dag = DaskVineDag(dsk)
+        dag = DaskVineDag(dsk, reconstruct=self.reconstruct, merge_size=self.merge_size)
+
+        if self.visualize:
```

Comment:
I think we want to leave the visualization to dask proper.

Reply:
Visualization option removed.